.circleci/config.yml

```diff
@@ -72,7 +105,7 @@ workflows:
       - publish-test:
           context: flambe-publisher
           requires:
-            - test
+            - examples_tests
```

Comment:
Until we don't figure out how CCIE can run the examples test, lets make `publish-test` dependent on `end2end_tests`

Reply:
Also this will allow to continue releasing automatically based on a gh tag.